deploy/olm/storageos/storageosoperator.0.0.10.clusterserviceversion.yaml

```diff
@@ -9,24 +9,93 @@ metadata:
     capabilities: Full Lifecycle
     categories: "Storage"
     alm-examples: '[{"apiVersion":"storageos.com/v1","kind":"StorageOSCluster","metadata":{"name":"example","namespace":"default"},"spec":{"secretRefName":"storageos-api","secretRefNamespace": "default"}}]'
-    description: An operator to setup and manage StorageOS cluster.
+    description: Cloud-native, persistent storage for containers.
 spec:
   # replaces: 
   displayName: Storageos Cluster Operator
   description: |
-    StorageOS transforms commodity server or cloud based disk capacity into enterprise-class storage to run persistent workloads such as databases in containers. The StorageOS Operator installs and manages StorageOS clusters.
+    StorageOS is a software-defined storage platform that transforms commodity
+    server or cloud based disk capacity into enterprise-class storage for
+    running persistent workloads such as databases in containers.
+
+    The StorageOS Cluster Operator installs and manages StorageOS within an
+    Openshift cluster. Cluster nodes may contribute local or attached disk-based
```

Comment:
Is this Openshift only or all K8S derivatives?

Reply:
Probably more targeted at Openshift at this point.

Reply:
All the development and tests, including e2e tests, were done on k8s.